.gitignore

```diff
@@ -25,3 +25,45 @@ pom.xml.asc
 
 .hgignore
 .hg/
+.DS_STORE
```

Comment:
Do these have to be copied to the root .gitignore? I think it would be easier to maintain if we let each example define its own .gitignore

Reply:
done